gatsby-node.js

```diff
@@ -157,8 +182,8 @@ exports.createPages = ({graphql, actions}) => {
 	}
 
 	return new Promise((resolve, reject) => {
-		const markdownPage = path.resolve('src/templates/markdown.js');
-		const jsonPage = path.resolve('src/templates/json.js');
+		const markdownPage = path.resolve('./src/templates/markdown.js');
+		const apiPage = path.resolve('./src/templates/apidoc.js');
```

Comment:
How about apiDocPage to be more specific?
```suggestion
		const apiDocPage = path.resolve('./src/templates/apidoc.js');
```

Reply:
It's fixed.